contracts/ChainlinkPricer.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+import {AggregatorInterface} from "./interfaces/AggregatorInterface.sol";
+import {OracleInterface} from "./interfaces/OracleInterface.sol";
+import {OpynPricerInterface} from "./interfaces/OpynPricerInterface.sol";
+import {Ownable} from "./packages/oz/Ownable.sol";
+import {SafeMath} from "./packages/oz/SafeMath.sol";
+
+contract ChainLinkPricer is OpynPricerInterface, Ownable {
+    using SafeMath for uint256;
+
+    /// @notice the opyn oracle address
+    OracleInterface public oracle;
+
+    /// @notice the aggregator for an asset
+    mapping(address => address) public assetAggregator;
+
+    /// @notice the mapping of price
+
+    constructor(address _oracle) public {
```

Comment:
Will be better to add aggregator address as argument here, and use one pricer for one feed, easier to understand and less permissions as we will not need the onlyOwner in this case.

Reply:
Thats a good idea, will change it ;)